R/pkgs/hospitalization/R/hospdeath.R

```diff
@@ -0,0 +1,127 @@
+
+build_hospdeath_par <- function(p_hosp, p_death, p_vent, p_ICU, data_filename, scenario_name, target_geo_ids,
+                                p_hosp_type="gamma",
+                                time_hosp_pars = c(1.23, 0.79), 
+                                time_ICU_pars = c(log(10.5), log((10.5-7)/1.35)),
+                                time_vent_pars = c(log(10.5), log((10.5-8)/1.35)),
+                                time_death_pars = c(log(11.25), log(1.15)), 
+                                time_disch_pars = c(log(11.5), log(1.22)),
+                                time_ICUdur_pars = c(log(17.46), log(4.044)),
+                                end_date = "2020-04-01",
+                                cores=8,
+                                root_out_dir='hospitalization') {
+  
+  n_sim <- length(list.files(data_filename))
+  print(paste("Creating cluster with",cores,"cores"))
+  doParallel::registerDoParallel(cores)
+  
+  print(paste("Running over",n_sim,"simulations"))
+  dat_final <- foreach::foreach(s=seq_len(n_sim), .packages=c("dplyr","readr","data.table","tidyr")) %dopar% {
+    create_delay_frame <- function(X, p_X, data_, X_pars, varname) {
+      X_ <- rbinom(length(data_[[X]]),data_[[X]],p_X)
+      # nonzero_mask <- X_ > 0
+      nonzero_mask <- rep(TRUE,length(X_))
```

Comment:
Nonzero mask should be removed now that we aren't doing that filtration. Not required for this stage